PS.g4

```diff
@@ -250,7 +295,7 @@ accent:
     accent_symbol
     L_BRACE base=expr R_BRACE;
 
-atom: (LETTER | SYMBOL | accent) subexpr? | NUMBER | DIFFERENTIAL | mathit | PLACEHOLDER;
+atom: (LETTER | SYMBOL | GREEK_LETTER | accent) subexpr? | NUMBER | E_NOTATION | DIFFERENTIAL | mathit | PLACEHOLDER;
```

Comment:
@tragessere does `SYMBOL` still exist?

Reply:
It does, but looking at this again it should probably be moved outside the parentheses and over to the right. I don't think it makes sense for the symbols to have a subscript anymore